src/applications/personalization/profile-2/sass/profile-2.scss

```diff
@@ -82,3 +82,37 @@ nav.profile-side-nav {
     margin: unset;
   }
 }
+
+
+.headerNameWrapper {
+  width: 60%;
+  h2 {
+    text-transform: capitalize;
+  }
+  @media (max-width: $medium-screen) {
+    width: 100%;
+    text-align: center;
+
+    h3 {
+      margin-top: 0;
+    }
+  }
+}
+
+.profileServiceBadge {
+  max-height: 108px;
+}
+
+.headerContentWrapper {
+  width: 100%;
+  @media (max-width: $medium-screen) {
+    flex-direction: column;
+    width: 100%;
+  }
+}
+
+.profileHeading{
+  @media (max-width: $medium-screen) {
```

Comment:
I think you can just add the class [`medium-screen:vads-u-display--none`](https://design.va.gov/utilities/flexbox) to achieve this instead of doing it via custom styling! 🙂

Reply:
same deal. but, yeah, the class Kelson shared would hide the section at medium or wider screens since the classes operate on a mobile-first basis https://design.va.gov/utilities/display#responsive-prefixes